ioctl/newcmd/account/account.go

```diff
@@ -182,21 +183,24 @@ func PrivateKeyFromSigner(signer, password string) (crypto.PrivateKey, error) {
 }
 
 // GetAccountMeta gets account metadata
-func GetAccountMeta(addr string) (*iotextypes.AccountMeta, error) {
-	conn, err := util.ConnectToEndpoint(config.ReadConfig.SecureConnect && !config.Insecure)
+func GetAccountMeta(addr string, client ioctl.Client) (*iotextypes.AccountMeta, error) {
+	var endpoint string
+	var insecure bool
+	apiServiceClient, err := client.APIServiceClient(ioctl.APIServiceConfig{
+		Endpoint: endpoint,
+		Insecure: insecure,
```

Comment:
here Endpoint = "", Insecure = false?

Reply:
no, these values should get by client.Config() , for next grpc dial

Reply:
when I made the comment, the code is like this:
```
var endpoint string
var insecure bool
```

Reply:
so i think it's correct to say:
yes, fix by getting these values by client.Config() , for next grpc dial